include/libhal-sensor/multi/bmp180.hpp

```diff
@@ -24,13 +24,12 @@ namespace hal::sensor {
 /**
  * @brief Driver for the BMP180 barometric pressure/temperature sensor
  */
-
 class bmp180
 {
 public:
   /**
-   * @brief Oversampling rate defined with the mode names as well as sample
-   * fetch times.
+   * @brief Stores the oversampling rate which is defined with the mode names as
+   * well as sample fetch times.
```

Comment:
```suggestion
   * @brief Defines the oversampling rate with names containing the mode as
   * well as sample fetch times.
```

I wouldn't consider an enum class as "storing" stuff. I think this is what you meant in your comment.